src/routes/namespaces/[namespace]/workflows/[workflow]/[run]/history/index.svelte

```diff
@@ -1,5 +1,31 @@
 <script lang="ts">
-  import WorkflowHistory from '$lib/pages/workflow-history.svelte';
+  import { page } from '$app/stores';
+  import { eventViewType } from '$lib/stores/event-view';
+
+  import WorkflowRunLayout from '$lib/layouts/workflow-run-layout.svelte';
+  import WorkflowHistoryLayout from '$lib/layouts/workflow-history-layout.svelte';
+  import WorkflowHistoryFeed from '$lib/pages/workflow-history-feed.svelte';
+  import WorkflowHistoryJson from '$lib/pages/workflow-history-json.svelte';
+  import WorkflowHistoryCompact from '$lib/pages/workflow-history-compact.svelte';
+
+  import PageTitle from '$lib/components/page-title.svelte';
+
+  const workflow = $page.params.workflow;
+
+  const views = {
+    feed: WorkflowHistoryFeed,
+    compact: WorkflowHistoryCompact,
+    json: WorkflowHistoryJson,
+  };
+  $: view = views[$eventViewType] ?? WorkflowHistoryFeed;
 </script>
 
-<WorkflowHistory />
+<PageTitle title={`Workflow History | ${workflow}`} url={$page.url.href} />
+<WorkflowRunLayout cancelEnabled>
+  <WorkflowHistoryLayout>
+    <!-- <svelte:fragment slot="timeline">
+    <EventHistoryTimelineContainer />
+    </svelte:fragment> -->
```

Comment:
Is this supposed to be commented out?

Reply:
yeah for the time being. I leave it commented out so we know where it goes once it's ready to flip on.